templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul>
+  <li><a href="{% url 'taxi:index' %}">Home page</a></li>
+  <li><a href="{% url 'taxi:index' %}">Manufacturers</a></li>
+  <li><a href="{% url 'taxi:index' %}">Cars</a></li>
+  <li><a href="{% url 'taxi:index' %}">Drivers</a></li>
```

Comment:
The links for 'Manufacturers', 'Cars', and 'Drivers' all point to the home page. Update these links to point to the correct URLs for each respective page using the `{% url %}` template tag with the appropriate view names.